src/components/Dialog/Dialog.tsx

```diff
@@ -277,9 +277,17 @@ export default class Dialog extends PureComponent<DialogProps, DialogState> {
   getContainer() {
     const { containerSelector } = this.props;
     if (!containerSelector) {
+      console.error("Dialog: containerSelector is not provided or is falsy. Dialog may not be correctly positioned.");
```

Comment:
This should be reverted - it's fine if containerSelector is undefined or null
The error should be thrown only if it's not null, but such element doesn't exist on the page - as you did below